client/webserver/site/src/js/order.ts

```diff
@@ -257,14 +259,21 @@ export default class OrderPage extends BasePage {
     tmpl.status.textContent = OrderUtil.matchStatusString(m)
 
     const setCoin = (pendingName: string, linkName: string, coin: Coin) => {
+      const formatCoinID = (cid: string) => {
+        if (cid.startsWith(coinIDTakerFoundMakerRedemption)) {
+          const makerAddr = cid.substring(coinIDTakerFoundMakerRedemption.length)
+          return intl.prep(intl.ID_TAKER_FOUND_MAKER_REDEMPTION, { makerAddr: makerAddr })
+        }
+        return cid
+      }
```

Comment:
Front end is taking on a new role and I'm not certain it's necessary. Why not just set the coin ID to `fmt.Sprintf("redeemed by %s", makerAddr)` in `FindRedemption`?  It's still a constant length string for `(Driver),DecodeCoinID`. Or you could leave `FindRedemption` as you have it, and format the `redeemed by` string in `DecodeCoinID`, which we be more amenable to internationalization.

Reply:
> Why not just set the coin ID to fmt.Sprintf("redeemed by %s", makerAddr) in FindRedemption?

I considered that, but we'd still need to take that same role when extracting `address` to build a link to Etherscan (see `const ethExplorers` definition), hence I though it is better to define this CoinID format **explicitly** instead of relying on the exact phrasing **implicitly** (which by the way changed [as per your suggestion](https://github.com/decred/dcrdex/pull/1978#discussion_r1040278273): `${makerAddr} redeemed` -> `redeemed by ${makerAddr}` - which would have broken Etherscan-url-building code if not adjusted accordingly),

> Or you could leave FindRedemption as you have it, and format the redeemed by string in DecodeCoinID, which we be more amenable to internationalization.

and that same reasoning I outlined above also applies here.

Reply:
Yeah, the explorer link builder is a pickle.  If we think of a better solution later, it shouldn't be a problem to change it since this encoding exception exists only on the client.

Reply:
Well, aside from adding a new field to the `core` package's `type Coin struct` and modifying `NewCoin` somehow